api/util/constants.js

```diff
@@ -30,9 +30,12 @@ const consoleColors = {
   defaultColor: '\x1b[0m',
 };
 
+const PASSWORD_RESET_EXPIRATION = 60 * 60 * 12;
```

Comment:
add a comment above this variable about what this means
```js
// this is set to 12 hours by:
// 60 seconds per minute * 60 minutes per hour * 12 hours
```

Reply:
done